.github/workflows/kotlin.yml

```diff
@@ -201,49 +217,54 @@ jobs:
             --continue
           restore-cache-key: build-logic
           write-cache-key: main-build-artifacts
+          failure-path-upload: '**/build/reports/tests/*[tT]est'
+          failure-upload-name: 'unit-tests-report'
```

Comment:
Here is where we specify `failure-path-upload` (what to upload when there is a failure on the job).